### ISSUE TITLE
Лоты, которых нет в продаже и ножи

### ISSUE THREAD

**Issue by rsunru** [Maxx53/SCMBot#248]
в бот не добавляются ножи (из cs go), выскакивает ошибка "Nothing found".
+ хорошо бы было иметь возможность добавлять лоты, которых нет в продаже (чтобы скупить лот, как только его кто-то выставит на продажу), при добавлении ссылки на такой лот таже самая ошибка

**Comment by Maxx53** [comment]
Пользуемся поиском
https://github.com/Maxx53/SCMBot/issues/160

**Comment by rsunru** [comment]
да, спасибо, как раз только что нашел это, надо было сначала поискать. извиняюсь)

**Comment by rsunru** [comment]
только одно уточнение: лоты, которых нет в продаже надо добавлять во вкладку "Новые поступления"?

**Comment by Maxx53** [comment]
Для большей эффективности лучше в обе вкладки.

**Comment by rsunru** [comment]
понял, спасибо

**Comment by rsunru** [comment]
никак не пойму, перестало добавляться в вкладку новые поступления (название и ссылка не перепутаны), с другими лотами также. на вкладке "страница предметов" те же лоты добавляются
https://www.dropbox.com/s/trqevk3hvckay7s/Screenshot%202015-01-13%2009.56.15.png?dl=0

**Comment by rsunru** [comment]
возникло несколько вопросов:
-графа "название" обязательна должно соответствовать лоту или это для себя?
-если разместить ссылку на лот, которого нет на тп, только в вкладке "новые поступления", купится ли лот?
-если разместить ссылку на лот, которого нет на тп, только в вкладке "страница предмета", купится ли лот?

**Comment by Maxx53** [comment]
Вкладка "страница предмета" читает страницу по указанной ссылке, имя игнорируется.
Вкладка "новые поступления" ищет совпадение по имени, ссылка игнорируется.

**Comment by rsunru** [comment]
Еще такой вопрос: при большом количестве лотов (20+ например) цена в правой графе обновляется медленнее, и чем больше лотов, тем медленнее. Хотя задержка везде 3000. Это срабатывает защита, или бот все равно быстро сканирует, но не отображает этого? И если это защита, то есть ли какие-нибудь способы для быстрого сканирование 20-30 итемов? Заранее спасибо за ответ)
